features/step_definitions/skin_steps.rb

```diff
@@ -134,3 +134,34 @@
 Then /^I should see a different accent color on the dashboard and work meta$/ do
   step %{I should see "#header .icon, #dashboard ul, #main dl.meta {background: blue; border-color:blue;}" within "style"}
 end
+
+Then /^the cache of the skin on "([^\"]*)" should expire after I save the skin$/ do |title| 
+  skin = Skin.find_by_title(title)
+  orig_cache_key = skin_cache_value(skin)
+  visit edit_skin_path(skin) 
+  fill_in("CSS", with: "#random { text-decoration: blink;}")
+  click_button("Update") 
+  assert orig_cache_key != skin_cache_value(skin), "Cache key #{orig_cache_key} matches #{skin_cache_value(skin)}." 
+end 
+
+Then(/^the cache of the skin on "(.*?)" should not expire after I save "(.*?)"$/) do |arg1, arg2|
+  skin = Skin.find_by_title(arg1)
+  save_me = Skin.find_by_title(arg2)
+  orig_skin_key = skin_cache_value(skin)
+  orig_save_me_key = skin_cache_value(save_me)
+  visit edit_skin_path(save_me)
+  fill_in("CSS", with: "#random { text-decoration: blink;}")
+  click_button("Update")
+  assert orig_save_me_key != skin_cache_value(save_me), "Cache key #{orig_save_me_key} matches #{skin_cache_value(save_me)}"
+  assert orig_skin_key == skin_cache_value(skin),"Cache key #{orig_skin_key} does not match #{skin_cache_value(skin)}"
+end
+
+Then(/^the cache of the skin on "(.*?)" should expire after I save a parent skin$/) do |arg1|
+  skin = Skin.find_by_title(arg1)
+  orig_skin_key = skin_cache_value(skin)
+  parent = Skin.find_by_title("Parent")
+  parent_id=SkinParent.where(:child_skin_id => skin.id).last.parent_skin_id
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=`.